api/test/integration/test_cluster_endpoints.tavern.yaml

```diff
@@ -566,7 +566,7 @@ marks:
 
 stages:
 
-  - name: Request {master-node}
+  - name: Request (master-node)
```

Comment:
Side note: this has been solved in `4.5.2` in [this](https://github.com/wazuh/wazuh/issues/18317#event-10105336801) issue